exo.jcr.component.core/src/main/java/org/exoplatform/services/jcr/impl/core/WorkspaceImpl.java

```diff
@@ -462,16 +462,22 @@ public void move(String srcAbsPath, String destAbsPath) throws ConstraintViolati
             (NodeData)srcParentNode.getData(), nodeTypeManager, session.getTransientNodesManager(), true,
             triggerEventsForDescendants, session.maxDescendantNodesAllowed);
 
-      srcNode.getData().accept(initializer);
-      PlainChangesLog changes = new PlainChangesLogImpl(initializer.getAllStates(), session);
+      try {
+         srcNode.getData().accept(initializer);
+      } catch (Exception e) {
+         LOG.warn(e.getMessage());
```

Comment:
IMO, this can't be accepted as solution since this will catch any type of exception event security exception, connection interruption exception, foreign key exception, database dropped exception .... In all conditions, we ask to commit changes on database.

Reply:
> Let's dig more to check why we have empty properties and how to not consider this kind of data corrupted.

no problem, I hope you are satisfied with the effort I have done on this task :) .

Reply:
Yep, of course, thanks a lot. In fact, it's a very complex use case, the swallowed exception [here](https://github.com/exoplatform/ecms/blob/516b97e706bc9a7741042e1905482c0c7db1093c/core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/rightclick/manager/DeleteManageComponent.java#L403-L408) is : 

`javax.jcr.ReferentialIntegrityException: Can not delete node org.exoplatform.services.jcr.datamodel.QPath@83cb1944 ([]:1[http://www.jcp.org/jcr/1.0]system:1[http://www.jcp.org/jcr/1.0]versionStorage:1[]4c941d5f7f0001015eaa3d1fdd2ab9cc:1[]4:1) (4c95540e7f000101204df26107df7eab). It is currently the target of a REFERENCE property []:1[]sites:1[]dw:1[]web contents:1[]foldera:1[]webcontentA:1[http://www.exoplatform.com/jcr/publication/1.1/]liveRevision:1
`

So there is multiple issues here:
1/ Avoid swallow exception and log errors in addition to displaying error message to user
2/ Review copy/paste process in WebContent nodetype to avoid referencing versions from original Node
3/ Apply a migration process or a workaround for old data having a bad JCR structure